utils/near-cache/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+use cached::{Cached, SizedCache};
+use std::hash::Hash;
+use std::sync::Mutex;
+
+pub struct MyCache<K, V> {
```

Comment:
This name is not optimal. Could to change to `Cache` instead (or some more descriptive name)?

Reply:
@bowenwant1996
Maybe `NearLRUCache`

Reply:
I'd just `LRUCache` -- there's no near specific thing here. And Rust is not C, so there's no need to add perfixes to names just to avoid clashes.